pkg/session/session.go

```diff
@@ -1426,7 +1427,197 @@ func (s *session) GetTiDBTableValue(name string) (string, error) {
 
 var _ sqlexec.SQLParser = &session{}
 
+const (
+	coreSQLToken     = 1 << 0
+	bypassSQLToken   = 1 << 1
+	isSelectSQLToken = 1 << 2
+)
+
+var keySQLToken = map[string]int{
+	"select": isSelectSQLToken,
+	"from":   coreSQLToken, "insert": coreSQLToken, "update": coreSQLToken, "delete": coreSQLToken, "replace": coreSQLToken, "analyze": coreSQLToken,
+	"execute": coreSQLToken, // ignore `prepare` statement because its content will be parsed later
+	"explain": bypassSQLToken, "desc": bypassSQLToken}
+
+func approxSQLTokenCnt(sql string) (tokenCnt int64) {
```

Comment:
I don't understand what the return value `tokenCnt` is used for? Can you add some comments?